src/lib/LoginFile.js

```diff
@@ -21,8 +21,8 @@ class LoginFile {
 
     static calculateQrPosition() {
         return {
-            x: 69,
-            y: 287,
+            x: 138,
+            y: 536,
```

Comment:
So, this is not `2 * 287` on purpose?

Reply:
Every element except the logo and the date moved up, because of spacing changes.